package.json

```diff
@@ -2,32 +2,45 @@
   "name": "ps-tree",
   "version": "1.1.0",
   "description": "Get all children of a pid",
-  "homepage": "http://github.com/indexzero/ps-tree",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/indexzero/ps-tree.git"
-  },
-  "dependencies": {
-    "event-stream": "~3.3.0"
-  },
   "license": "MIT",
+  "homepage": "http://github.com/indexzero/ps-tree#readme",
+  "repository": "github:indexzero/ps-tree",
+  "bugs": {
+    "url": "https://github.com/indexzero/ps-tree/issues",
+    "email": "charlie.robbins@gmail.com"
+  },
   "author": "Charlie Robbins <charlie.robbins@gmail.com>",
   "contributors": [
-    {
-      "name": "Zhuohuan LI",
-      "url": "https://github.com/zixia",
-      "email": "zixia@zixia.net"
-    }
+    "Zhuohuan LI <zixia@zixia.net> (https://github.com/zixia)",
+    "Simone Primarosa <simonepri@outlook.com> (https://github.com/simonepri)"
+  ],
+  "keyboards": [
+    "ps-tree",
+    "ps",
+    "tree",
+    "ppid",
+    "pid"
+  ],
+  "main": "index.js",
+  "bin": {
+    "ps-tree": "./bin/ps-tree.js"
+  },
+  "files": [
+    "bin",
+    "index.js"
   ],
-  "directories": {
-    "test": "test"
+  "engines": {
+    "node": ">= 0.10"
   },
   "scripts": {
     "_comment": "https://github.com/gotwarlost/istanbul#usage-on-windows",
     "test": "istanbul cover node_modules/tape/bin/tape test/test.js test/direct.js",
     "coverage": "npm test && istanbul check-coverage --statements 100 --functions 100 --lines 100 --branches 100",
     "codeclimate": "cross-env CODECLIMATE_REPO_TOKEN=84436b4f13c70ace9c62e7f04928bf23c234eb212c0232d39d7fb1535beb2da5 codeclimate < coverage/lcov.info"
   },
+  "dependencies": {
+    "event-stream": "~3.3.0"
```

Comment:
It's _unclear_ from `this` PR where `event-stream` is being used. 
Perhaps consider adding detail to the PR description as to why/where it is required. 🤔

Reply:
event-stream is already in the package.json
I just reordered the keys